client/app/app.css

```
@@ -1,6 +1,8 @@
 @import "tailwindcss";
 @import "tw-animate-css";
 
+@custom-variant arrows-fit (@container (min-height: 72px));
```

Stylelint (17.13.0)

[error] Unexpected unknown at-rule "`@custom-variant`" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)